test/TensorFlowRuntime/test_remote.swift

```diff
@@ -0,0 +1,33 @@
+// RUN: %target-run-simple-swift  | %FileCheck %s
```

Comment:
do we still need this new test suite, given https://github.com/apple/swift/blob/1d166ce255b14d2e513b3fdd66de64c97953632f/test/TensorFlowRuntime/tensor.swift#L557?

I feel it's a bit redundant, but don't have a strong preference.

Reply:
Yes, we could skip this. Was thinking of making a more direct test for remote mode. If there is a way to check if something is remote, it might be worth having this test. Otherwise, I will drop this PR.